pystiche/papers/ulyanov_et_al_2016/loss.py

```diff
@@ -0,0 +1,165 @@
+from typing import Union, Optional, Sequence, Any, Dict
+from collections import OrderedDict
+import torch
+from pystiche.enc import Encoder, MultiLayerEncoder
+from pystiche.ops import (
+    MSEEncodingOperator,
+    GramOperator,
+    MultiLayerEncodingOperator,
+)
+from torch.nn.functional import mse_loss
+from pystiche.loss import MultiOperatorLoss
+from .utils import ulyanov_et_al_2016_multi_layer_encoder
+
+
+class UlyanovEtAl2016MSEEncodingOperator(MSEEncodingOperator):
+    def __init__(
+        self, encoder: Encoder, score_weight: float = 1e0
+    ):
+        super().__init__(encoder, score_weight=score_weight)
+        self.loss_reduction = "mean"
```

Comment:
Since the default `loss_reduction` is `"mean"` there is not reason to set it explicitly.